crosswalk/index.html

```diff
@@ -815,19 +815,37 @@ <h3><a href="https://www.w3.org/2021/a11y-discov-vocab/latest/#accessibilityFeat
                         <p>The resource includes static page markers, such as those identified by the doc-pagebreak role [[DPUB-ARIA-1.1]].</p>
                         <p>This value is most commonly used with ebooks for which there is a statically paginated equivalent, such as a print edition, but it is not required that the page markers correspond to another work. The markers may exist solely to facilitate navigation in purely digital works.</p></td>
                     <td>
-                        <p><a href="https://www.w3.org/2021/a11y-discov-vocab/latest/#pageBreakMarkers"
-                                    ><code>pageBreakMarkers</code></a><br /> (formerly
+                        <p><a href="https://www.w3.org/2021/a11y-discov-vocab/latest/#pageBreakMarkers"><code>pageBreakMarkers</code></a><br /> (formerly
                             <code>printPageNumbers</code>)</p>
                     </td>
                     <td>
-                        <p><a href="https://ns.editeur.org/onix/en/196/19">List: 196; Code 19</a>: Print-equivalent page
-                            numbering</p>
+                        <p><a href="https://ns.editeur.org/onix/en/196/19">List: 196; Code 19</a>: Print-equivalent page numbering (Source: print book)</p>
+                     </td>
+                    <td>-</td>
+					<td>-</td>
+					<td>-</td>
+                    <td>-</td>
+				</tr>
+                 <tr>
+                    <td>
+                        <p>Provides a unique identifier for the source of the page break markers in an EPUB publication.</p>
+                     </td>
+                    <td>
+                        <p><a href="https://www.w3.org/publishing/a11y/page-source-id/#pageBreakSource"><code>pageBreakSource</code></a></p>
+                        <p>value of <code>none</code> implys digital-only edition</p>
```

Comment:
I'd say "indicates" here, not implies.

```suggestion
                        <p>value of <code>none</code> indicates digital-only edition</p>
```